python/pyspark/sql/column.py

```diff
@@ -250,11 +250,39 @@ def __iter__(self):
         raise TypeError("Column is not iterable")
 
     # string methods
+    _rlike_doc = """ Return a Boolean :class:`Column` based on a regex match.\n
```

Comment:
I just manually tried after replacing `\n` to a newline with `make clean` and `make html`. It seems fine to use a newline (not `\n`). Could you double check and replace it if I was not wrong?